wandb/internal/internal_api.py

```diff
@@ -29,7 +29,7 @@
 from wandb import __version__
 from wandb.old.core import wandb_dir, Error
 from wandb import env
-#from wandb.git_repo import GitRepo
+from wandb.old.git_repo import GitRepo
```

Comment:
It's confusing how we have git_repo in old and in internal.  We should likely use the one in `internal` unless there's something I'm missing.  We may even be able to get rid of the one in "old" as everything in that directory should eventually be removed.  I guess this is fine for now, but eventually we'll factor old out of this codebase so wanted to flag it.

Reply:
old git_repo was around to support code sharing between cli-og during the transition.   If we want to access git_repo from internal process and user process then please move internal/git_repo or somehow consolidate under wandb/lib/